packages/fiori/src/ShellBar.hbs

```diff
@@ -107,6 +109,7 @@
 					icon="sap-icon://search"
 					data-ui5-text="Search"
 					data-ui5-notifications-count="{{notificationsCount}}"
+					data-ui5-stable="searchButton"
```

Comment:
**[suggestion (non-blocking)](https://conventionalcomments.org/#labels):**

```suggestion
					data-ui5-stable="search"
```

Reply:
This one at least should hav button, because I think the search itself has "search" and this is just the magnifying glass icon.